ludwig/modules/convolutional_modules.py

```diff
@@ -841,75 +855,71 @@ def __init__(
             img_width: int,
             first_in_channels: int,
             out_channels: int,
-            stride: int,
+            stride: int = 1,
             batch_norm_momentum: float = 0.9,
             batch_norm_epsilon: float = 0.001,
-            projection_shortcut: Optional[nn.LudwigModule] = None
+            projection_shortcut: Optional[LudwigModule] = None
     ):
-        super().__init__()
+        """ Resnet blocks used for ResNet34 and smaller.
 
-        self.projection_shortcut = projection_shortcut
-
-        self.norm1 = nn.BatchNorm2d(
-            num_features=first_in_channels,
-            eps=batch_norm_epsilon,
-            momentum=batch_norm_momentum,
-        )
-
-        self.activation1 = get_activation('relu')
+        stride: A single int specifying the stride of the first convolution.
+            The last convolution will have stride of 1.
+        """
+        super().__init__()
+        self._input_shape = (first_in_channels, img_height, img_width)
 
         self.conv1 = Conv2DLayerFixedPadding(
-            img_height=img_height,
-            img_width=img_width,
-            in_channels=first_in_channels,
-            out_channels=out_channels,
-            kernel_size=3,
-            stride=stride,
-        )
+            img_height, img_width, first_in_channels, out_channels,
```

Comment:
Recommend using keyword args here, as using args alone can sometimes result in difficult to debug errors when interfaces change.

Reply:
Done!